pyvera/__init__.py

```diff
@@ -490,6 +495,10 @@ def vera_device_id(self):
         """The ID Vera uses to refer to the device."""
         return self.device_id
 
+    @property
+    def should_poll(self):
```

Comment:
I can certainly add the should_poll changes to the locks if you'd like that in this pull.